modules/weboramaRtdProvider.js

```diff
@@ -921,7 +977,18 @@ export function isValidProfile(profile) {
     return false;
   }
 
-  return Object.values(profile).every((field) => isArray(field) && field.every(isStr));
+  return Object.values(profile).every(
+    (field) => isArray(field) && field.every(isStrOrNumber)
```

Comment:
That is the most important change: because sometimes the profile could contain not only strings but numbers, it end return false in some scenarions